alma/views/templates/hook/_partials/displayPayment_deferred.tpl

```diff
@@ -48,7 +48,7 @@
         <div class="row">
             <div class="col-xs-12">
                 <p class="payment_module">
-                    <a href="{$option.link}" class="alma-fragments-deferred {$almaButton} {$iconDisplay}">
+                    <a href="{$option.link}" class="alma-inpage-ps16 {$almaButton} {$iconDisplay}" id="payment-option-{$option.paymentOptionKey}">
```

Comment:
In page is not yet on deferred payment

Reply:
The option is disabled for deferred in back. I have just anticipate the integration in front :)